src/esphomelib/application.h

```diff
@@ -663,7 +665,13 @@ class Application {
   MakeESP32HallSensor make_esp32_hall_sensor(const std::string &name, uint32_t update_interval = 15000);
 #endif
 
+#ifdef USE_RTC_COMPONENT
+time::RTCComponent *make_rtc_component(const std::string &tz);
```

Comment:
Indent the `make_rtc_component` and `make_sntp_component` component lines please (but not the preprocessor macros)